### PR TITLE
Add slug field description

### PR DESCRIPTION
**Pull request by krzysztofwolski**
Added slug field description for our conventions category.

### CODE REVIEW

**Review by fowczarek — APPROVED** [review]

File: docs/developer/slug-fields.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Slug fields
+---
+
+## Introduction
+Slug is a short label for an object. Main reason for using slugs is ability to encorporate them in URLs. Translated names can contain all utf-8 signs, which could lead to URLs which could not be written for people with different keyboard layout.
```
Comment (karolkielecki) [inline docs/developer/slug-fields.mdx:6]: A slug is a short label that can explain an object or page's content. The main reason for using a slug is the ability to incorporate it to URLs. Translated names can contain all UTF-8 signs—this allow for URLs to be written by people with different keyboard layouts.

Comment (karolkielecki) [inline docs/developer/slug-fields.mdx:6]:
```suggestion
A slug is a short label that can explain an object or page's content. The main reason for using a slug is the ability to incorporate it to URLs. Translated names can contain all UTF-8 signs—this allow for URLs to be written by people with different keyboard layouts.
```

File: docs/developer/slug-fields.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Slug can contain:
+- letters
+- numbers
+- underscores
+- hyphens
+
+By convention slugs in Saleor are not translatable, so the users using different languages could use the same product URLs.
```
Comment (karolkielecki) [inline docs/developer/slug-fields.mdx:14]: Slugs are not translatable in Saleor. When using different languages users should apply the same product URLs regardless of the language.

Comment (karolkielecki) [inline docs/developer/slug-fields.mdx:14]:
```suggestion
Slugs are not translatable in Saleor. When using different languages users should apply the same product URLs regardless of the language.
```

File: docs/developer/slug-fields.mdx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+## Example
+
+As example we would like to create product called "Żółta ćma" (Yellow moth). Expected slug for such product would be `Zolta-cma`.
```
Comment (karolkielecki) [inline docs/developer/slug-fields.mdx:18]: For example: we would like to create a product called "Żółta ćma" (Yellow moth). The expected slug for such product would be `Zolta-cma`.

Comment (karolkielecki) [inline docs/developer/slug-fields.mdx:18]:
```suggestion
For example: we would like to create a product called "Żółta ćma" (Yellow moth). The expected slug for such product would be `Zolta-cma`.
```

**Review by karolkielecki — APPROVED** [review]

**Pull request merged**